include/otf2xx/definition/detail/calling_context_impl.hpp

```diff
@@ -110,24 +98,24 @@ namespace definition
                 return parent_.get() != nullptr;
             }
 
-            calling_context_impl* parent() const
+            auto parent() const
```

Comment:
Sad that this can't be the definition or at least `weak_ref` it actually is. Wasn't able to find a way to do it...